contrib/build-wine/prepare-wine.sh

```diff
@@ -70,8 +79,8 @@ KEYSERVER_PYTHON_DEV="hkp://keys.gnupg.net"
 gpg --no-default-keyring --keyring $KEYRING_PYTHON_DEV --keyserver $KEYSERVER_PYTHON_DEV --recv-keys $KEYLIST_PYTHON_DEV
 for msifile in core dev exe lib pip tools; do
     echo "Installing $msifile..."
-    wget "https://www.python.org/ftp/python/$PYTHON_VERSION/win32/${msifile}.msi"
-    wget "https://www.python.org/ftp/python/$PYTHON_VERSION/win32/${msifile}.msi.asc"
+    wget -nc "https://www.python.org/ftp/python/$PYTHON_VERSION/win32/${msifile}.msi"
```

Comment:
Why not `download_if_not_exist` too?

Reply:
The -nc prevents wget from downloading if the file already exists, but cannot be used with -O for the output file or else it will download every time, making the caching feature useless. I didn't run into that problem using -nc with these files because we didn't need to specify an output file.